.github/workflows/ci.yml

```diff
@@ -0,0 +1,71 @@
+name: Compile and run tests
+
+on: [push, pull_request]
+
+jobs:
+  test_standard_build:
+    container:
+      image: fedora:latest
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install libcomps dependencies
+        run: |
+          sudo dnf install -y gcc make cmake python-devel bzip2-devel expat-devel libxml2-devel xz-devel
+
+      - name: Install documentation dependencies
+        run: |
+          sudo dnf install -y doxygen python3-sphinx
+
+      - name: Install test dependencies
+        run: |
+          sudo dnf install -y check check-devel
+
+      - name: Compile
+        run: |
+          mkdir build
+          cd build/
+          cmake ../libcomps -DPYTHON_DESIRED=3
+          make
+
+      - name: Build documentation
+        run: |
+          cd build/
+          make docs
+          make pydocs
+
+      - name: Test
+        run: |
+          cd build/
+          make tests
+          make test
+
+  test_python_build:
```

Comment:
Could you update this job to be identical to the createrepo_c one, except for necessary differences?

Reply:
Yes